result/neighborsv4.yaml

```diff
@@ -0,0 +1,43 @@
+openapi: 3.1.0
+
+info:
+  title: IPv4 Neighbors models
+  version: ^0.0.0
+
+components:
+  schemas:
+    Neighborsv4.States.Request:
+      description: >-
+        The request to retrieve IPv4 Neighbor state (ARP cache entries) for device network interface(s).
+      type: object
+      properties:
+        ethernet_names:
+          description: >-
+            The names of device Ethernet interfaces for which Neighbors will be retrieved.
+            If no names are specified then the results will contain Neighbors for all Ethernet interfaces.
```

Comment:
The names of Ethernet interfaces for which Neighbor state (ARP cache entries) will be retrieved.
If no names are specified then the results will contain Neighbor state (ARP cache entries)  for all available Ethernet interfaces.